File: fix-navigation-links.js
import fs from 'fs';

const filePath = './app/components/Navigation.tsx';
let content = fs.readFileSync(filePath, 'utf8');

// Fix all Link semicolons
content = content.replace(/<Link;/g, '<Link');

fs.writeFileSync(filePath, content);
console.log('Fixed all Link semicolons in Navigation.tsx');

File: fix_navigation_links.py
#!/usr/bin/env python3
import re

def fix_navigation_links():
    """Fix all href attributes to to attributes in Navigation.tsx"""
    file_path = '/workspace/app/components/Navigation.tsx'
    
    with open(file_path, 'r', encoding='utf-8') as f:
        content = f.read()
    
    # Replace all href="/..." with to="/..."
    content = re.sub(r'href="(/[^"]*)"', r'to="\1"', content)
    
    with open(file_path, 'w', encoding='utf-8') as f:
        f.write(content)
    
    print("Fixed all href attributes to to attributes in Navigation.tsx")

if __name__ == "__main__":
    fix_navigation_links()